test/tektonbundles_test.go

```diff
@@ -53,8 +53,9 @@ var bundleFeatureFlags = requireAnyGate(map[string]string{
 	"enable-api-fields":         "alpha",
 })
 
-var resolverFeatureFlags = requireAnyGate(map[string]string{
+var resolverFeatureFlags = requireAllGates(map[string]string{
 	"enable-bundles-resolver": "true",
```

Comment:
do we to set enable-bundles-resolver to true when enable-api-fields is already set to beta?

Reply:
I think we do as beta are for [apis fields validation ](https://github.com/tektoncd/pipeline/blob/fb292df20b8797619698ae578f31e39eafa6694f/pkg/apis/pipeline/v1/taskref_validation.go#L36) while the [`enable-bundles-resolver` is for the specific resolver](https://github.com/tektoncd/pipeline/blob/fb292df20b8797619698ae578f31e39eafa6694f/pkg/resolution/resolver/bundle/resolver.go#L106) that's used.